### PR TITLE
Venetian bruh

### PR DESCRIPTION
**Pull request by trinkledink**
The Venetian Republic has been controlled by the Patrician caste for hundreds of years. Their rule is represented by the Patrician Supremacy modifier you get at game start. It prevents you from forming Italy. Removing it requires abolishing the Patrician caste either through the decision or resolving the disaster.

As the world begins to change, springtime of the people-esque conditions will instead kick off a unique disaster where the player will have a choice whether to preserve Patrician rule, reform it, or abolish it. You will be given the choice between a Positivist Intelligentsia agitator employed to reform the caste and a Radical agitator of a non-patrician interest group looking to abolish it entirely. Your support of one over the other will give you access to more characters of agreeing ideologies throughout the disaster while you decide how to deal with the Italian and Greek nationalists,  Aragonese influence in the peninsula, and the tumult that other events bring.

Starting buildings and PM changes too.

### CODE REVIEW

**Comment by MadJamez** [comment]
@trinkledink Whats the difference between this and the other PR?

**Comment by trinkledink** [comment]
> @trinkledink Whats the difference between this and the other PR?

This is more complete and everything is consolidated into fewer commits because I thought the other branches were having problems because of your comments. 

I added a couple more events, balanced out the ticker, added a weekly pulse that ticks up/down depending on your government, and added a tooltip to make that clear to the player. Some other niche scenarios and loopholes were addressed as well, like exiling the opposition agitator is disabled.

File: events/dvg_venice.txt [hunk 1]
```diff
@@ -0,0 +1,1879 @@
+﻿namespace = dvg_ven
+
+# starting patrician event
+dvg_ven.1 = {
```
Comment (MadJamez) [inline events/dvg_venice.txt:4]: Small nitpick. But would be nice if Greek culture was also clickable in startup event. 
image

Comment (trinkledink) [reply events/dvg_venice.txt:4]: done. see bruh commit.

Comment (MadJamez) [reply events/dvg_venice.txt:4]: Its not working ingame. Still need to scope it in event
```
cu:greek= {
    save_scope_as = greek_culture
}
```

Comment (trinkledink) [reply events/dvg_venice.txt:4]: okay, done. Anything else?

**Review by MadJamez — APPROVED** [review]

**Pull request merged**